src/dask_histogram/layers.py

```diff
@@ -0,0 +1,15 @@
+from dask.layers import DataFrameTreeReduction
+
+
+class MockableDataFrameTreeReduction(DataFrameTreeReduction):
+    def mock(self):
+        return MockableDataFrameTreeReduction(
+            name=self.name,
+            name_input=self.name_input,
+            npartitions_input=1,
```

Comment:
I'm surprised it's this simple, but good!

I believe that we are requiring that dask-histogram be importable on the scheduler when we do this. That might in theory have side-effects, but sounds reasonable. We could avoid it by having a `__reduce__` which recreates itself as a plain DataFrameTreeReduction after deserialisation.

Can we please get some test coverage of this.

Reply:
I'm not really sure what a meaningful test of this would be. You just want something to check that the number of input partitions for `.mock()` really is 1 or something like that?

Reply:
Yes, exactly. 
I guess we should have pytest-cov running here, so we can tell at least that the code is being hit.